harness/typeCoercion.js

```diff
@@ -231,10 +271,8 @@ function testNotCoercibleToNumber(test) {
   // ToNumber: Symbol -> TypeError
   testPrimitiveValue(Symbol("1"));
 
-  if (typeof BigInt !== "undefined") {
-    // ToNumber: BigInt -> TypeError
-    testPrimitiveValue(BigInt(0));
-  }
+  // ToNumber: BigInt -> TypeError
+  testPrimitiveValue(0n);
```

Comment:
are we just throwing this away and assuming this file will only be used with BigInt?

Reply:
We need to decide one way or another. In some places in this harness file, BigInt literals are used unconditionally. This check here was pointless, since the file would not parse without BigInt support already. Additionally, the `BigInt` feature is required for this harness file according `harness/features.yml`.

This was also a concern when my `String.prototype.indexOf` type coercion test required BigInt support back in #1200 and #1221. I'm open to discussing more versatility in this regard.